ui/app/components/clients/dashboard.js

```diff
@@ -2,7 +2,7 @@ import Component from '@glimmer/component';
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
-import { isSameMonth } from 'date-fns';
+import { isSameMonth, compareAsc } from 'date-fns';
```

Comment:
would it make sense to pull these out into a util? so both dashboard and current can use them? And if we're adding to charts in 1.11 then each chart could access the function too?

Reply:
I think with the change to the isAfter I'm now using the method on date-fns and it's pretty clean. Thoughts?